yew-ui/src/components/attendants.rs

```diff
@@ -65,6 +74,8 @@ pub struct AttendantsComponentProps {
     #[prop_or_default]
     pub email: String,
 
+    pub e2ee_enabled: bool,
```

Comment:
This is awesome, in a follow up MR we can enable/disable encryption via a URL parameter